Hebrew payment status messages

ybelenko opened the pull request:
#### `PelecardPaymentStatus`

New class is responsible for handling payment status messages. Right now it contains `HE`(158 messages) and `EN`(173 messages) translations. I've decided to set `HE` locale as default, because API uses Hebrew as default language for redirect payment page. This class returns english message as a fallback, when hebrew translation doesn't exist or missing.

`PelecardPaymentStatus` is fully covered by unit tests.

#### Usage

```php
use Pelecard\PelecardPaymentStatus;

$status = new PelecardPaymentStatus('010', 'HE');
$status->getMessage();
// output:
// תוכנית הופסקה עפ"י הוראת המפעיל (ESC) או COM PORT לא ניתן לפתיחה (WINDOWS).
```

Commit bda4829 fixes indentation of my previous PR, sorry for that.

Closes #10

ybelenko committed:
Change indentation for consistency

ybelenko committed:
Add class for handling payment status messages

ybelenko committed:
Add unit tests for new class

reissr commented on src/Pelecard/PelecardPaymentStatus.php:
```diff
@@ -0,0 +1,423 @@
+
+  /* @var array Payment status messages, english version */
+  const MESSAGES_EN = [
+    '000' => 'Permitted transaction.',
```
Thanks (again) for your great contribution!
Why not using associative array here? It will also be easier to use on the get method, don't you think?
```
const $messages = [
  '000' => [
    'en' => 'fsdfd',
    'he' => 'כדכג',
  ],
];
```

Not really sure which one is easier to maintain, but as I know Pelecard - changes are rare and not languages will really be added, so we only enjoy more simplicity on the get method.

ybelenko replied:
You're right. I made it like this, because I used Notepad++ and regular expressions to convert whole excel table to single array with one click. I can overwrite it as you want manually, not a big deal.

reissr replied:
I'll merge it right after you make that change. Thanks!

ybelenko committed:
Update hebrew texts

getMessage method has been slightly refactored.
MESSAGES_HE and MESSAGES_EN merged into one associative array for
convenience.

ybelenko committed:
Update tests fixture

ybelenko committed:
Fix typo

ybelenko commented:
There are missing english texts for statuses:
- `301` - עבר זמן השהייה המותר בדף הסליקה.
- `302` - התשלום בוצע אך אתר העסק לא זמין לקבלת התשובה
- `303` - אתר העסק לא זמין
- `555` - לקוח לחץ על כפתור ביטול בדף התשלום
- `800` - שגיאה כללית - UPAY
- `801` - משתמש איננו משתמש UPAY
- `802` - שגיאה בפרטי תשלום ב- UPAY
- `803` - סוג תשלום אינו נתמך (UPAY)
- `804` - סוג טרנזאקציה לא נתמך (UPAY)
- `805` - חסרים פרטי משתמש הכרחיים של UPAY
- `806` - העסקה אינה עסקת J4
